packages/dm-core-plugins/src/mermaid-plugin/MermaidWrapper.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+// import mermaid from 'mermaid'
+//
+// mermaid.initialize({
+//     startOnLoad: true,
+//     theme: 'default',
+//     securityLevel: 'loose',
+//     themeCSS: `
+//     g.classGroup rect {
+//       fill: #282a36;
+//       stroke: #6272a4;
+//     }
+//     g.classGroup text {
+//       fill: #f8f8f2;
+//     }
+//     g.classGroup line {
+//       stroke: #f8f8f2;
+//       stroke-width: 0.5;
+//     }
+//     .classLabel .box {
+//       stroke: #21222c;
+//       stroke-width: 3;
+//       fill: #21222c;
+//       opacity: 1;
+//     }
+//     .classLabel .label {
+//       fill: #f1fa8c;
+//     }
+//     .relation {
+//       stroke: #ff79c6;
+//       stroke-width: 1;
+//     }
+//     #compositionStart, #compositionEnd {
+//       fill: #bd93f9;
+//       stroke: #bd93f9;
+//       stroke-width: 1;
+//     }
+//     #aggregationEnd, #aggregationStart {
+//       fill: #21222c;
+//       stroke: #50fa7b;
+//       stroke-width: 1;
+//     }
+//     #dependencyStart, #dependencyEnd {
+//       fill: #00bcd4;
+//       stroke: #00bcd4;
+//       stroke-width: 1;
+//     }
+//     #extensionStart, #extensionEnd {
+//       fill: #f8f8f2;
+//       stroke: #f8f8f2;
+//       stroke-width: 1;
+//     }`,
+//     fontFamily: 'Fira Code',
+// })
+
+export default class MermaidWrapper extends React.Component {
+  // componentDidMount() {
+  //     mermaid.contentLoaded()
+  // }
+
+  render() {
+    // eslint-disable-next-line react/prop-types
+    // return <div className="mermaid">{this.props.chart}</div>
+    return <div>Mermaid is not working</div>
```

Comment:
Has it never worked, or did this PR break it?

Reply:
Right now, mermaid plugin is not used in main (not imported in the example app):
![image](https://user-images.githubusercontent.com/69512295/221505408-a13409a5-efc1-4da6-bb17-72a4e79480a8.png)


Tried to add it back in main, and it also failed there.

 
 the mermaid plugin did work at some point (a few months ago). I experienced some strange dependency issues that I could not figure our